tests/OmniSharp.Tests/UpdateBufferFilterFacts.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNet.Mvc;
+using Microsoft.CodeAnalysis.CSharp;
+using OmniSharp.Filters;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
```

Comment:
and I was proud of myself for making sure it was alphabetical cause you mentioned sorting the other time too! Where are the coding guidelines yo! :smile: